packages/components/src/suggestions/index.jsx

```diff
@@ -159,11 +161,14 @@ class Suggestions extends Component {
 	}
 
 	render() {
-		const { query } = this.props;
+		const { query, className } = this.props;
 		const showSuggestions = this.getSuggestionsCount() > 0;
+		const containerClass = classnames( {
+			[className]: className,
+		} );
 
 		return (
-			<div>
+			<div className={ containerClass } >
```

Comment:
It doesn't seem like we need `classnames` for anything, we can just use the prop (or `undefined`), right?

```suggestion
			<div className={ className } >
```

Reply:
If you think that's better, yeah.

Reply:
I wonder if getting rid of the outer div somehow would be a better solution. We can still pass the optional class if we wanted:

```
const containerClass = classnames( 'suggestions__wrapper', {
    [className]: className,
} );
```

I sense this is also more common in the wild (passing an "additional" class vs an outer/enclosing one)?

Another (better?) option, keep the outer div and use `suggestions` as the parent class:

```
const containerClass = classnames( 'suggestions', {
    [className]: className,
} );
```

Reply:
> I wonder if getting rid of the outer div somehow would be a better solution.

I agree with this assessment :+1:  The outer `<div />` is largely useless: it's currently unstyled, and plainly wraps around the `showSuggestions` check (which is arguably also a bit weird to have at the top level, but that's for another discussion), and the `suggestions__wrapper` `<div />`.

> We can still pass the optional class if we wanted:
> 
> ```
> const containerClass = classnames( 'suggestions__wrapper', {
>     [className]: className,
> } );
> ```
> 
> I sense this is also more common in the wild (passing an "additional" class vs an outer/enclosing one)?

Yep :+1: 

> Another (better?) option, keep the outer div and use `suggestions` as the parent class:
> 
> ```
> const containerClass = classnames( 'suggestions', {
>     [className]: className,
> } );
> ```

Let's not keep two immediately nested `<div />` (where the outer one doesn't have any other children), that's just confusing. However, it's worth considering renaming the `suggestions__wrapper` class to `suggestions`.

Reply:
Just one minor note on syntax, let's just do

```js
const containerClass = classnames( 'suggestions__wrapper', className );
```

[`classnames`](https://www.npmjs.com/package/classnames) is smart enough to do the right thing (even if `className` is falsey).

Reply:
>  However, it's worth considering renaming the `suggestions__wrapper` class to `suggestions`.

Do you think `suggestions` may be too broad of a term for a CSS class? Do we have some convention for these (e.g. a prefix maybe, like `wp-calypso-suggestions`), or am I missing something?

Reply:
`suggestions` seems fine.